Makefile

```diff
@@ -38,7 +38,7 @@ generate: ## Generate code and manifests
 	$(Q)go generate ./...
 
 # Static tests.
-.PHONY: test test-unit verify
+.PHONY: test test-unit verify build bin/k8s
```

Comment:
`bin/k8s` is a real target though.

Reply:
Sure, I'm only interested in ensuring this target always gets run, so maybe .FORCE would be more applicable here. I typically sway towards using .PHONY as it's more explicit imo. I don't have a strong opinion either way.

Reply:
I realize I'm not entirely sure whether .FORCE would be relevant here - the `make bin/k8s` target creates local `./bin/k8s` executable, so adding that as a .PHONY target would make more sense. What we could pivot towards instead is creating a `clean` target that removes this file beforehand, and mark that target as a prerequisite to the `bin/k8s` target. I'd guess the .PHONY implementation is a bit more efficient but it's likely negligible at best.